components/camel-rest-openapi/src/main/docs/rest-openapi-component.adoc

```diff
@@ -0,0 +1,249 @@
+[[rest-openapi-component]]
+= REST OpenApi Component
+= REST OpenApi Component
+*Since Camel 3.1*
+
+
+*Since Camel 3.1*
+
+// HEADER START
+*Only producer is supported*
+// HEADER END
+
+The REST OpenApi* configures rest producers from
+http://swagger.io/[OpenApi] (Open API) specification document and
```

Comment:
We could link to https://www.openapis.org/

Reply:
fixed accordingly, thanks!